tests/vm.rs

```diff
@@ -195,6 +197,12 @@ impl VMIsolate {
                                 v.length as usize,
                             );
                         },
+                        DartTypedDataType::Usize => {
+                            let _ = from_buf_raw(
+                                v.values as *mut usize,
+                                v.length as usize,
+                            );
+                        },
```

Comment:
As I mentioned, there is nothing as `usize` in dartvm, so there is no need for it here.
```suggestion
```